demo/demo.py

```diff
@@ -105,8 +104,6 @@ def init(loop):
     app.router.add_route('GET', '/call_ajax', call_ajax, name='call_ajax')
 
     # templates error handlers
-    app.router.add_route('GET', '/mako_exc', test_mako_exc,
```

Comment:
Could you revert ``app.router.add_route('GET', '/mako_exc', test_mako_exc`` changes in demo app? I want to have both: ``mako`` and ``jinja2`` templates there to make sure that both working and exceptions generated inside template engine have meaningful representation.